.github/workflows/lint.yml

```diff
@@ -0,0 +1,31 @@
+
+name: Lint
```

Comment:
I think the linting step runs in the `studio` build, but it might be nice to have some faster feedback & a more focused workflow for linting the various projects.